packages/content-gen/docs/MESHY_API.md

```
@@ -0,0 +1,383 @@
+    ▼ model_urls.glb, texture_urls[0].base_color
+rigging (input_task_id OR model_url + texture_image_url)
+    │
+    ▼ result.rigged_character_glb_url
+animations (rig_task_id + action_id)
+    │
+    ▼ result.animation_glb_url
+```
+
+---
+
+*Last Updated: 2025-01-15*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)